tfworker/authenticators/aws.py

```diff
@@ -106,6 +109,7 @@ def _session_state_args(self):
 
     @property
     def account_id(self):
+        """ Nothing uses this?  OK to remove? """
```

Comment:
I think this is a relic from the early version, if someone wanted this information, they should get it using a terraform data source, seems okay to remove